### ISSUE TITLE
Change keys for GCI/ejection

### ISSUE THREAD

**Issue by colingeniet** [NikolaiVChr/flightgear-saab-ja-37-viggen#108]
F6 is ejection, F7-9 is GCI. As you can imagine, this results in unwanted ejections when using GCI.

Any suggestion on a better choice of keybindings?

**Comment by JMaverick16** [comment]
F8>F10 maybe?

**Comment by Rudolf339** [comment]
Idk what the F-16 uses, but the MiG-21 uses f5-7 for GCI, it would be nice to make it standard. For the GCI I would make it say shift+f8 so you can't really hit it y accident

**Comment by colingeniet** [comment]
F10 toggles the menubar, that's not a keybinding I want to mess with, even if most people generally don't use it.

Here's the current keys for some of the other aircrafts:
- f16 and f14: ejection F6, gci ctrl 1-3
- m2000: ejection ctrl-E x3, gci F6-F8
- mig21: no ejection, gci F5-F7

Viggen uses ctrl 1-8 to select pylons. I think these keybindings are fairly obsolete and not really realistic (irl, AJS uses a knob to select weapon type, JA can select pylons on the TI, and has 'shortcuts' on the throttle). Maybe they could be deleted to have the same control scheme as f16/f14.

**Comment by Rudolf339** [comment]
Yeah, same as the f-14/16 sounds the best. Though making ejection shift+F6 would be nice, just in case

**Comment by NikolaiVChr** [comment]
Yeah, I agree that the pylon keybindings are obsolete.

**Comment by colingeniet** [comment]
Pylon keybindings are removed, GCI is now ctrl 1-3

> making ejection shift+F6

I like the idea of a 'safer' ejection shortcut, but since F6 is used by several aircrafts, I think it would be better to change it consistently, and thus discuss it more publicly.

**Comment by colingeniet** [comment]
Reopening the issue because I just realised that the keybindings `ctrl-2` and `ctrl-3` are completely broken for me:
- `ctrl-2` seems to do the same as ctrl-1 (and it's not because I made a copy/paste error in the keybindings file)
- `ctrl-3` seems to generate `ESC` (it will give the 'Exit Flightgear' prompt dialog, or close the active dialog if any)

Since it has to do with keyboard input, I suspect it may also be OS dependent. I'm using linux (X11).
It would be helpful if other persons could test this, including using other OS.

The GCI keybindings set one of the three properties in `/instrumentation/gci` each, it's the easiest way to check if they work.
